src/sidebar/components/tag-editor.js

```diff
@@ -207,17 +224,38 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
           );
         })}
       </ul>
-      <input
-        list={datalistId}
-        onInput={handleOnInput}
-        onKeyPress={handleKeyPress}
-        onKeyUp={handleKeyUp}
-        ref={inputEl}
-        placeholder="Add tags..."
-        className="tag-editor__input"
-        type="text"
-      />
-      {suggestionsList()}
+      <span
+        id={tagEditorId}
+        className="tag-editor__combobox-wrapper"
+        ref={closeWrapperRef}
+        role="combobox"
+        aria-controls={`${tagEditorId}-datalist`}
```

Comment:
The `aria-controls` attribute should be set on the input field rather than the wrapper, per https://www.w3.org/TR/wai-aria-1.1/#combobox and the examples at https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

Reply:
yep, I saw that! good catch